### ISSUE TITLE
watchcat: support for multiple IPs broken

### ISSUE THREAD

**Issue by raineth** [openwrt/packages#7148]
Maintainer: @nunojpg 
Environment: all, OpenWrt 18.06.1 r7258-5eb055306f

Description:
Watchcat is written to support multiple space-separated addresses: https://github.com/openwrt/packages/blob/2e0cb11fe0bc9129e71252eafb5967eaa5bda2d8/utils/watchcat/files/watchcat.sh#L55

...but it doesn't work.  I think the behavior of the shell changed at some point in the past, because the quotes around "$pinghosts" are forcing multiple space-separated addresses to be passed to ping as 1 argument which fails.  This used to work on some ancient version of OpenWRT that I just upgraded from.

Removing the quotes from "$pinghosts" causes the shell to split multiple space-separated hosts and ping them properly, counting a response from any address as "up."  This is pretty clearly the intention, and with the change space-separated addresses in system.@watchcat[0].pinghost*s* work properly.

tl;dr:  
please replace the line above with
```shell
	for host in $pinghosts
```
